### ISSUE TITLE
Auto Play next downloaded video

### ISSUE THREAD

**Issue by Nikhar14** [polymorphicshade/NewPipe#301]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/NewPipe/issues) or [closed](https://github.com/polymorphicshade/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/polymorphicshade/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Auto Play next downloaded video. Also please add play next button in the player.

Steps:
Go to downloads play any video from the downloaded video list. After the video is finished, we have to manually select the next video to play...it doesn't auto Play next. Also, if some video is playing, back button restarts the video but pressing it again should take me to the previous video. Similarly play next button should be there to skip the present video and play the next in the queue

I am not a tech sevy. Please if somebody could better describe and create a new issue (and if it's related to upstream?)

### Why do you want this feature?

Makes the download video playing feature convenient and seamless. It is a very basic feature among the video players 

### Why ist the feature relevant to this fork?

Because it is related to downloads

**Comment by heartlog** [comment]
Most of the feature enhancement request are ignored...Well it's same for built in player.
Why don't you see it yourself ? #30 

That's what I have to say 🙂.

**Comment by polymorphicshade** [comment]
@heartlog I don't ignore any feature requests, but I also have a very small amount of free time I can dedicate to this project. Any new features I work on will end up in the tubular branch.

@Nikhar14 This would be great, but this feature shouldn't be specific to my fork. Request it upstream, and if they explicitly reject it, post here and I'll re-open the issue.

**Comment by polymorphicshade** [comment]
I just realized this only makes sense in the built-in player specific to my fork. 😅
Re-opening this issue for now.

**Comment by Nikhar14** [comment]
> I just realized this only makes sense in the built-in player specific to my fork. 😅 Re-opening this issue for now.

Yeah...I raised it upstream, and they closed it. It's most probably applicable to this fork as it is related to built in player. Thanks
